Failed to build on my MacOS VM

```
In file included from third-party/tbb/include/tbb/../oneapi/tbb/task.h:22:
third-party/tbb/include/tbb/../oneapi/tbb/detail/_task.h:221:13: error: aligned deallocation function of type 'void (void *, std::align_val_t) noexcept' is only available on macOS 10.14 or newer
    virtual ~task() = default;
            ^
third-party/tbb/include/tbb/../oneapi/tbb/parallel_for.h:66:8: note: in defaulted destructor for 'tbb::detail::d1::task' first required here
struct start_for : public task {
       ^
third-party/tbb/include/tbb/../oneapi/tbb/parallel_for.h:66:8: note: in implicit destructor for 'tbb::detail::d1::start_for<tbb::detail::d1::blocked_range<long long>, tbb::detail::d1::parallel_for_body_wrapper<(lambda at elf/compress.cc:72:49), long long>, const tbb::detail::d1::auto_partitioner>' first required here
third-party/tbb/include/tbb/../oneapi/tbb/parallel_for.h:247:5: note: in instantiation of template class 'tbb::detail::d1::start_for<tbb::detail::d1::blocked_range<long long>, tbb::detail::d1::parallel_for_body_wrapper<(lambda at elf/compress.cc:72:49), long long>, const tbb::detail::d1::auto_partitioner>' requested here
    start_for<Range,Body,const auto_partitioner>::run(range,body,partitioner);
    ^
third-party/tbb/include/tbb/../oneapi/tbb/parallel_for.h:316:9: note: in instantiation of function template specialization 'tbb::detail::d1::parallel_for<tbb::detail::d1::blocked_range<long long>, tbb::detail::d1::parallel_for_body_wrapper<(lambda at elf/compress.cc:72:49), long long>>' requested here
        parallel_for(range, body, partitioner);
        ^
third-party/tbb/include/tbb/../oneapi/tbb/parallel_for.h:355:5: note: in instantiation of function template specialization 'tbb::detail::d1::parallel_for_impl<long long, (lambda at elf/compress.cc:72:49), const tbb::detail::d1::auto_partitioner>' requested here
    parallel_for_impl<Index,Function,const auto_partitioner>(first, last, static_cast<Index>(1), f, auto_partitioner());
    ^
elf/compress.cc:72:8: note: in instantiation of function template specialization 'tbb::detail::d1::parallel_for<long long, (lambda at elf/compress.cc:72:49)>' requested here
  tbb::parallel_for((i64)0, (i64)inputs.size(), [&](i64 i) {
       ^
third-party/tbb/include/tbb/../oneapi/tbb/detail/_task.h:221:13: note: if you supply your own aligned allocation functions, use -faligned-allocation to silence this diagnostic
    virtual ~task() = default;
            ^
In file included from elf/compress.cc:17:
In file included from third-party/tbb/include/tbb/parallel_for_each.h:17:
In file included from third-party/tbb/include/tbb/../oneapi/tbb/parallel_for_each.h:27:
third-party/tbb/include/tbb/../oneapi/tbb/parallel_for.h:66:8: error: aligned deallocation function of type 'void (void *, std::align_val_t) noexcept' is only available on macOS 10.14 or newer
struct start_for : public task {
       ^
third-party/tbb/include/tbb/../oneapi/tbb/parallel_for.h:66:8: note: in implicit destructor for 'tbb::detail::d1::start_for<tbb::detail::d1::blocked_range<long long>, tbb::detail::d1::parallel_for_body_wrapper<(lambda at elf/compress.cc:72:49), long long>, const tbb::detail::d1::auto_partitioner>' first required here
third-party/tbb/include/tbb/../oneapi/tbb/parallel_for.h:247:5: note: in instantiation of template class 'tbb::detail::d1::start_for<tbb::detail::d1::blocked_range<long long>, tbb::detail::d1::parallel_for_body_wrapper<(lambda at elf/compress.cc:72:49), long long>, const tbb::detail::d1::auto_partitioner>' requested here
    start_for<Range,Body,const auto_partitioner>::run(range,body,partitioner);
    ^
third-party/tbb/include/tbb/../oneapi/tbb/parallel_for.h:316:9: note: in instantiation of function template specialization 'tbb::detail::d1::parallel_for<tbb::detail::d1::blocked_range<long long>, tbb::detail::d1::parallel_for_body_wrapper<(lambda at elf/compress.cc:72:49), long long>>' requested here
        parallel_for(range, body, partitioner);
        ^
third-party/tbb/include/tbb/../oneapi/tbb/parallel_for.h:355:5: note: in instantiation of function template specialization 'tbb::detail::d1::parallel_for_impl<long long, (lambda at elf/compress.cc:72:49), const tbb::detail::d1::auto_partitioner>' requested here
    parallel_for_impl<Index,Function,const auto_partitioner>(first, last, static_cast<Index>(1), f, auto_partitioner());
    ^
elf/compress.cc:72:8: note: in instantiation of function template specialization 'tbb::detail::d1::parallel_for<long long, (lambda at elf/compress.cc:72:49)>' requested here
  tbb::parallel_for((i64)0, (i64)inputs.size(), [&](i64 i) {
       ^
third-party/tbb/include/tbb/../oneapi/tbb/parallel_for.h:66:8: note: if you supply your own aligned allocation functions, use -faligned-allocation to silence this diagnostic
struct start_for : public task {
       ^
third-party/tbb/include/tbb/../oneapi/tbb/parallel_for.h:66:8: error: aligned deallocation function of type 'void (void *, std::align_val_t) noexcept' is only available on macOS 10.14 or newer
struct start_for : public task {
       ^
third-party/tbb/include/tbb/../oneapi/tbb/parallel_for.h:66:8: note: in implicit destructor for 'tbb::detail::d1::start_for<tbb::detail::d1::blocked_range<long long>, tbb::detail::d1::parallel_for_body_wrapper<(lambda at elf/compress.cc:101:49), long long>, const tbb::detail::d1::auto_partitioner>' first required here
third-party/tbb/include/tbb/../oneapi/tbb/parallel_for.h:247:5: note: in instantiation of template class 'tbb::detail::d1::start_for<tbb::detail::d1::blocked_range<long long>, tbb::detail::d1::parallel_for_body_wrapper<(lambda at elf/compress.cc:101:49), long long>, const tbb::detail::d1::auto_partitioner>' requested here
    start_for<Range,Body,const auto_partitioner>::run(range,body,partitioner);
    ^
third-party/tbb/include/tbb/../oneapi/tbb/parallel_for.h:316:9: note: in instantiation of function template specialization 'tbb::detail::d1::parallel_for<tbb::detail::d1::blocked_range<long long>, tbb::detail::d1::parallel_for_body_wrapper<(lambda at elf/compress.cc:101:49), long long>>' requested here
        parallel_for(range, body, partitioner);
        ^
third-party/tbb/include/tbb/../oneapi/tbb/parallel_for.h:355:5: note: in instantiation of function template specialization 'tbb::detail::d1::parallel_for_impl<long long, (lambda at elf/compress.cc:101:49), const tbb::detail::d1::auto_partitioner>' requested here
    parallel_for_impl<Index,Function,const auto_partitioner>(first, last, static_cast<Index>(1), f, auto_partitioner());
    ^
elf/compress.cc:101:8: note: in instantiation of function template specialization 'tbb::detail::d1::parallel_for<long long, (lambda at elf/compress.cc:101:49)>' requested here
  tbb::parallel_for((i64)0, (i64)shards.size(), [&](i64 i) {
       ^
third-party/tbb/include/tbb/../oneapi/tbb/parallel_for.h:66:8: note: if you supply your own aligned allocation functions, use -faligned-allocation to silence this diagnostic
struct start_for : public task {
       ^
third-party/tbb/include/tbb/../oneapi/tbb/parallel_for.h:66:8: error: aligned deallocation function of type 'void (void *, std::align_val_t) noexcept' is only available on macOS 10.14 or newer
struct start_for : public task {
       ^
third-party/tbb/include/tbb/../oneapi/tbb/parallel_for.h:66:8: note: in implicit destructor for 'tbb::detail::d1::start_for<tbb::detail::d1::blocked_range<long long>, tbb::detail::d1::parallel_for_body_wrapper<(lambda at elf/compress.cc:120:49), long long>, const tbb::detail::d1::auto_partitioner>' first required here
third-party/tbb/include/tbb/../oneapi/tbb/parallel_for.h:247:5: note: in instantiation of template class 'tbb::detail::d1::start_for<tbb::detail::d1::blocked_range<long long>, tbb::detail::d1::parallel_for_body_wrapper<(lambda at elf/compress.cc:120:49), long long>, const tbb::detail::d1::auto_partitioner>' requested here
    start_for<Range,Body,const auto_partitioner>::run(range,body,partitioner);
    ^
third-party/tbb/include/tbb/../oneapi/tbb/parallel_for.h:316:9: note: in instantiation of function template specialization 'tbb::detail::d1::parallel_for<tbb::detail::d1::blocked_range<long long>, tbb::detail::d1::parallel_for_body_wrapper<(lambda at elf/compress.cc:120:49), long long>>' requested here
        parallel_for(range, body, partitioner);
        ^
third-party/tbb/include/tbb/../oneapi/tbb/parallel_for.h:355:5: note: in instantiation of function template specialization 'tbb::detail::d1::parallel_for_impl<long long, (lambda at elf/compress.cc:120:49), const tbb::detail::d1::auto_partitioner>' requested here
    parallel_for_impl<Index,Function,const auto_partitioner>(first, last, static_cast<Index>(1), f, auto_partitioner());
    ^
elf/compress.cc:120:8: note: in instantiation of function template specialization 'tbb::detail::d1::parallel_for<long long, (lambda at elf/compress.cc:120:49)>' requested here
  tbb::parallel_for((i64)0, (i64)inputs.size(), [&](i64 i) {
       ^
third-party/tbb/include/tbb/../oneapi/tbb/parallel_for.h:66:8: note: if you supply your own aligned allocation functions, use -faligned-allocation to silence this diagnostic
struct start_for : public task {
       ^
third-party/tbb/include/tbb/../oneapi/tbb/parallel_for.h:66:8: error: aligned deallocation function of type 'void (void *, std::align_val_t) noexcept' is only available on macOS 10.14 or newer
struct start_for : public task {
       ^
third-party/tbb/include/tbb/../oneapi/tbb/parallel_for.h:66:8: note: in implicit destructor for 'tbb::detail::d1::start_for<tbb::detail::d1::blocked_range<long long>, tbb::detail::d1::parallel_for_body_wrapper<(lambda at elf/compress.cc:154:49), long long>, const tbb::detail::d1::auto_partitioner>' first required here
third-party/tbb/include/tbb/../oneapi/tbb/parallel_for.h:247:5: note: in instantiation of template class 'tbb::detail::d1::start_for<tbb::detail::d1::blocked_range<long long>, tbb::detail::d1::parallel_for_body_wrapper<(lambda at elf/compress.cc:154:49), long long>, const tbb::detail::d1::auto_partitioner>' requested here
    start_for<Range,Body,const auto_partitioner>::run(range,body,partitioner);
    ^
third-party/tbb/include/tbb/../oneapi/tbb/parallel_for.h:316:9: note: in instantiation of function template specialization 'tbb::detail::d1::parallel_for<tbb::detail::d1::blocked_range<long long>, tbb::detail::d1::parallel_for_body_wrapper<(lambda at elf/compress.cc:154:49), long long>>' requested here
        parallel_for(range, body, partitioner);
        ^
third-party/tbb/include/tbb/../oneapi/tbb/parallel_for.h:355:5: note: in instantiation of function template specialization 'tbb::detail::d1::parallel_for_impl<long long, (lambda at elf/compress.cc:154:49), const tbb::detail::d1::auto_partitioner>' requested here
    parallel_for_impl<Index,Function,const auto_partitioner>(first, last, static_cast<Index>(1), f, auto_partitioner());
    ^
elf/compress.cc:154:8: note: in instantiation of function template specialization 'tbb::detail::d1::parallel_for<long long, (lambda at elf/compress.cc:154:49)>' requested here
  tbb::parallel_for((i64)0, (i64)shards.size(), [&](i64 i) {
       ^
third-party/tbb/include/tbb/../oneapi/tbb/parallel_for.h:66:8: note: if you supply your own aligned allocation functions, use -faligned-allocation to silence this diagnostic
struct start_for : public task {
       ^
5 errors generated.
make: *** [<builtin>: elf/compress.o] Error 1
builder for '/nix/store/kmd5j72dmfxk3pb4aj9by1p033s8nd66-mold-0.9.6.drv' failed with exit code 2
error: build of '/nix/store/kmd5j72dmfxk3pb4aj9by1p033s8nd66-mold-0.9.6.drv' failed
```

--- a/pkgs/development/tools/mold/default.nix
+++ b/pkgs/development/tools/mold/default.nix
@@ -17,7 +17,7 @@ stdenv.mkDerivation rec {
     owner = "rui314";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256:0mj258fy8l4i23jd6ail0xrrq3das7lmrf1brrr1591ahx4vjj14";
+    sha256 = "0mj258fy8l4i23jd6ail0xrrq3das7lmrf1brrr1591ahx4vjj14";
   };
 
   buildInputs = [ zlib openssl ];

